### PR TITLE
Add common metadata required for RedHat certification

### PR DESCRIPTION
**Pull request by shivamerla**

### CODE REVIEW

File: bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -3,7 +3,19 @@ kind: ClusterServiceVersion
 metadata:
   name: nim-operator-certified.v0.1.0
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
```
Comment (slu2011) [inline bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml:7]: For longer term, at one time we need to add arm64 support for grace / blackwell? I believe gpu operator already does that?

**Review by slu2011 — APPROVED** [review]

File: bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (shivamerla) [reply bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml:7]: yes, whenever NIMs are ready for arm64

File: deployments/container/Dockerfile [hunk 2]
```diff
@@ -23,10 +23,23 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 
 # Use distroless as minimal base image to package the manager binary
 FROM nvcr.io/nvidia/distroless/go:v3.1.2
+
+ARG VERSION="unknown"
```
Comment (visheshtanksale) [inline deployments/container/Dockerfile:27]: Should we be passing the release version?

**Review by visheshtanksale — APPROVED** [review]
LGTM

File: deployments/container/Dockerfile (hunk 2 above)
Comment (shivamerla) [reply deployments/container/Dockerfile:27]: yes, it is passed from the Makefile

**Pull request merged**